__tests__/app.test.js

```diff
@@ -116,3 +115,58 @@ describe('get articles',()=>{
   });
 
 })
+
+describe('GET /api/articles/:article_id/comments',()=>{
+  test("return 200 and comments by article's id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({body})=>{
+        expect(body.comments).toHaveLength(2);
+            body.comments.forEach((comment) => {
+              expect(comment).toHaveProperty("comment_id", expect.any(Number)),
+              expect(comment).toHaveProperty("body", expect.any(String)),
+              expect(comment).toHaveProperty("votes", expect.any(Number)),
+              expect(comment).toHaveProperty("author", expect.any(String)),
+              expect(comment).toHaveProperty("created_at", expect.any(String)),
+              expect(comment).toHaveProperty("article_id", expect.any(Number))
+            })
+      })
+  });
+  test("return 400 and message of incorrectly entered ID", () => {
+    return request(app)
+        .get("/api/articles/55abc/comments")
+        .expect(400)          
+        .then(({ body }) => {
+        expect(body.message).toBe('Not a number, please enter valid id')
+        })
+  });
+  test('return 200 and array of objects which is by default sorted by "created_at" in a DESC order', () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("return 200 and message that there are no comments associated with this article", () => {
+    return request(app)
+        .get("/api/articles/4/comments")
+        .expect(200)          
+        .then(({ body }) => {
+        expect(body.comments).toEqual([
+          {
+            message: 'There are no comments associated with this article'
+          }
+        ])
+        })
+  })
+  test("return 404 and message that an article does not exist if article ID is not in db", () => {
+    return request(app)
+        .get("/api/articles/5555/comments")
+        .expect(404)          
+        .then(({ body }) => {
+        expect(body.message).toBe('This article does not exist')
+        })
+  });
+})
```

Comment:
Some good testing here, I'd suggest keeping all 200 tests together and error handling tests below just to make it more readable.